foundry.toml

```diff
@@ -0,0 +1,12 @@
+[profile.default]
```

Comment:
You should consider adding a profile with the same optimization settings as the deployed contracts so that it'd be possible to run the tests against the same version

Reply:
Set the optimizer runs in the default profile to the same as the hardhat configuration.